RevenueCatUI/Runtime/RevenueCatUI.cs

```diff
@@ -0,0 +1,102 @@
+using System;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace RevenueCat.UI
+{
+    /// <summary>
+    /// Main interface for RevenueCat UI components.
+    /// Provides methods to present paywalls and customer center.
+    /// </summary>
+    public static class RevenueCatUI
+    {
+        /// <summary>
+        /// Presents a paywall configured in the RevenueCat dashboard.
+        /// </summary>
+        /// <param name="options">Options for presenting the paywall</param>
+        /// <returns>A PaywallResult indicating what happened during the paywall presentation</returns>
+        public static async Task<PaywallResult> PresentPaywall(PaywallOptions options = null)
```

Comment:
We haven't used `Task` or async/await in Unity before... Though it does make sense for these scenarios. We should just make sure we test it thoroughly and we make sure there are no incompatibilities 🙏